packages/amesos2/src/Amesos2_EpetraMultiVecAdapter_decl.hpp

```diff
@@ -152,6 +152,8 @@ namespace Amesos2 {
     /// Return \c true if this MV has constant stride between vectors on this node
     bool isConstantStride() const;
 
+    bool isGIDContiguous() const;
```

Comment:
Not used, was added and used before I updated the solver parameter list to pass IsContiguous setting; will remove